packages/react-app/src/components/bridge/FromToken.jsx

```diff
@@ -35,6 +36,10 @@ export const FromToken = () => {
   const smallScreen = useBreakpointValue({ base: true, lg: false });
   const [balanceLoading, setBalanceLoading] = useState(false);
 
+  useEffect(() => {
+    isCustomTokenAbsent && onOpen();
```

Comment:
Lets not open customTokenModal at all, we should directly `setToken` with the given information.

Reply:
Ahh. So you mean set the default STAKE if the asked token aint found, correct?

Reply:
Yes. So there's a callback `setToken` which takes input as token. 
Here token = { chainId, address }

only these two (chainId and address) are important, because the rest of the information will be fetched, and set as needed.

Reply:
and in case setToken fails we can call setDefaulToken which will set STAKE token.

Reply:
but when setToken does fail let's show a toast saying token not found or something.

Reply:
Perfectooo